safe/report/extractors/aggregate_result.py

```diff
@@ -57,10 +57,15 @@ def aggregation_result_extractor(impact_report, component_metadata):
     aggregation_summary_fields = aggregation_summary.keywords[
         'inasafe_fields']
     debug_mode = impact_report.impact_function.debug_mode
-    use_aggregation = bool(impact_report.impact_function.provenance[
-                               'aggregation_layer'])
-    if not use_aggregation:
-        return context
+    # use_aggregation = bool(impact_report.impact_function.provenance[
+    #                            'aggregation_layer'])
+
+    # Before removing the aggregation table if there is no aggregation used,
+    # make sure that no aggregation is treated differently with selecting
+    # only one area.
```

Comment:
@Gustry I need to ask you here, does `entire area` treated equally with ie choosing only `one selection area`?

Reply:
Yes, if you have one row only, the total will be the same. Let's check together.

Reply:
ah yes about this, I mean if we are selecting only one feature from the aggregation area, does it count as an aggregation area or does it count as no aggregation area? if it's stll count as an aggregation area than we are safe to remove the aggregation report

Reply:
Even with one feature in the aggregation layer, we have an aggregation layer, so yes, it counts as an aggregation area.